client/components/public/imports/HomeDonate.jsx

```diff
@@ -21,20 +22,22 @@ export default class HomeDonate extends Component {
                   Please Consider:<br/>
                 </p>
                 <ul>
-                  <li>Sponsoring a student or team that needs help to participate.</li>
-                  <li>Non-monetary support such as prizes or signing up as a volunteer!</li>
+                  <li><a href="mailto:millie@wwu.edu">Sponsoring</a> a student or team that needs help to participate.</li>
```

Comment:
Can you please replace the raw email address with an obfuscated/encoded one? This will hopefully reduce the number of bots that find Millie's email address and add her to spam lists.

Here's a nice site to create obfuscated emails:
http://www.wbwip.com/wbw/emailencoder.html

Would turn her address into `&#109;&#105;&#108;&#108;&#105;&#101;&#064;&#119;&#119;&#117;&#046;&#101;&#100;&#117;`

Be sure to still include the `mailto:` as seen in the example on the website.